courses/OfferedCourses.md

```diff
@@ -53,7 +53,7 @@ hide_table_of_contents: true
 | COMP-4150   |     Advanced and Practical Database Systems      | Offered     | Not Offered | Not Offered | 💛           |                       COMP-3150, COMP-3300                        |
 | COMP-4200   |          Mobile Application Development          | Not Offered | Offered     | Not Offered | 💛           |                       COMP-3150, COMP-3220                        |
 | COMP-4220   |            Agile Software Development            | Offered     | Not Offered | Not Offered | 💛           |                             COMP-3220                             |
-| COMP-4250   |      Big Data Analytics and Database Design      | Not Offered | Offered     | Not Offered | 💛           |                             COMP-3150                             |
+| COMP-4250   |      Big Data Analytics and Database Design      | Not Offered | Offered     | Not Offered | 💛💜         |                             COMP-3150                             |
```

Comment:
Source? Just want to be sure this is correct and not a typo.

Reply:
I talked to my advisor about required courses and she told me that I need it